.github/workflows/backport.yaml

```diff
@@ -0,0 +1,31 @@
+name: Automated Backporting
+
+on:
+  pull_request_target:
+    types:
+      - closed
+    branches:
+      - main
+
+jobs:
+  create_backport:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    steps:
+      - if: contains(github.event.pull_request.labels.*.name, '13.0.x')
+        name: 13.0.x Backport
+        uses: kiegroup/git-backporting@main
```

Comment:
Can we use a tag instead?

Reply:
Updated... but after my earlier comment about the GH action convention, this one doesn't do that and just provides `[v4.2.0](https://github.com/kiegroup/git-backporting/releases/tag/v4.2.0)` :sweat_smile: